source/linux/How_to_Guides/Target/How_to_boot_quickly.rst

```diff
@@ -513,6 +513,45 @@ In the `&main_uart0` node, connect the GPIO by adding
 
     test-gpios = <&main_gpio0 39 GPIO_ACTIVE_HIGH>;
 
+Update the :file:`drivers/tty/serial/8250/8250_omap.c` driver. Add the following section before the `omap8250_probe` function:
+
+.. code-block:: C
```

Comment:
Please normalize the indentation of these new codeblocks.

https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace